rover/source/docs/rover-failsafes.rst

```diff
@@ -45,7 +45,7 @@ This failsafe is triggered if the vehicle stops receiving `heartbeat messages <h
 
 - set :ref:`FS_GCS_ENABLE <FS_GCS_ENABLE>` to "1" to enable this failsafe
 - if :ref:`FS_ACTION <FS_ACTION>` is "1", the vehicle will :ref:`RTL <rtl-mode>` to home, if "2" the vehicle will :ref:`Hold <hold-mode>`
-- use the transmitter's mode switch to re-take control of the vehicle in :ref:`Manual <manual-mode>` (or any other mode)
+- you must use the transmitter's mode switch to re-take control of the vehicle in :ref:`Manual <manual-mode>` (or any other mode), or change modes via the GCS iself, if communication is re-established.
```

Comment:
it's an existing issue but the pilot could also change the mode from the gcs.. but I suspect most users will figure this out anyway

Reply:
not sure what you mean....what is the existing issue? and it says that they pilot can change it out of the gcs fs mode with the gcs itself if communication is re-established with the gcs...is this incorrect?